lib/cdo/rails_env_config.rb

```diff
@@ -0,0 +1,21 @@
+# Load a minimal dummy Rails app for env defaults if one isn't already loaded.
+# This provides
+unless defined?(Rails) && Rails.application
+  # Skip Sass-plugin railtie because it depends on ActionController.
+  Sass::RAILS_LOADED = true
```

Comment:
Is this a list of exceptions that's likely to grow over time as our application gets more complicated?  I started one of those here a year ago and... well, I'm not thrilled about it:

https://github.com/code-dot-org/code-dot-org/blob/a6707d00127e579cd9b702160a58c7616aee689a/apps/webpack.js#L9-L21

Reply:
I hope this list doesn't grow- I actually consider this to be a missing-require 'bug' in the Sass-Rails Railtie (though it's definitely rare for a `Rails::Application` to be created without `ActionController::Base` already loaded, I don't think the Rails API forbids it). I could file a fix upstream and track the issue, but for now this was a quick workaround.